shimmingtoolbox/cli/b1shim.py

```diff
@@ -15,28 +16,24 @@
 
 @click.command(context_settings=CONTEXT_SETTINGS)
 @click.option('--b1map', 'fname_b1_map', required=True, type=click.Path(exists=True),
-              help="Path to B1 nifti as returned by st_dicom_to_nifti.")
-@click.option('--mask', 'fname_mask', type=click.Path(exists=True), required=False,
-              help="3D nifti file used to define the static spatial region to shim. The coordinate system should be the"
-                   " same as ``b1map``'s coordinate system.")
-@click.option('--cp', 'fname_cp_weights', type=click.Path(exists=True), required=False,
-              help="json file containing CP weights. See config/cp_mode.json for example.")
+              help="Complex 3D B1+ map.")
+@click.option('--mask', 'fname_mask', type=click.Path(exists=True), required=False, help="3D boolean mask.")
 @click.option('--algo', 'algorithm', type=int, default=1, show_default=True,
-              help="Number from 1 to 3 specifying which algorithm to use for B1 optimization.\n"
-                   "1 - Optimization aiming to reduce the coefficient of variation (CoV) of the resulting B1+ field.\n"
-                   "2 - Magnitude least square optimization targeting a specific B1+ value. Target value required.\n"
-                   "3 - Maximizes the minimum B1+ value for better efficiency.")
-@click.option('--target', 'target', type=float, required=False, help="Target B1+ value used by algorithm 2 in nT/V.")
+              help="Number specifying the B1+ shimming algorithm:\n"
+                   "1 - Reduce B1+ coefficient of variation.\n"
+                   "2 - Target a specified B1+ value. Target value required.\n"
+                   "3 - Maximize minimum B1+ for higher signal.")
+@click.option('--target', 'target', type=float, required=False, help="B1+ value (nT/V) targeted by algorithm 2.")
```

Comment:
Type should either be click.int or click.choice. I usually go for 'choice' when you have specific options the user needs to input.

Reply:
In the [click documentation](https://click.palletsprojects.com/en/8.0.x/options/) they use type=float or type=int in their examples. I don't understand why we should avoid it here @po09i

Reply:
Maybe I was not clear sorry. We can use `float` when it makes sense. However, in this case, we are choosing between 3 options (`choice` makes more sense since it restricts what the user can input and we don't need to error check, click does it). `Int` also makes more sense than `float` but we need to manually do the error checking.

Reply:
So you're referring to the `algo` option not the `target` one right ? In this case it makes more sense

Reply:
Oups! yes I'm referring to algo 😅